examples/web/notification.html

```diff
@@ -5,10 +5,16 @@
 <head>
     <meta charset="utf-8" />
     <title></title>
+
+    <script>
```

Comment:
This is used by OpenFin example and is the template for its toasts.  The onNotificationMessage is invoked on load and the message provided in the notification options is provided as a parameter.